### DISCUSSION TITLE
v0.75.1

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#6570] [Announcements]
### Improvements

* Added flag `--no-migrate` to optionally suppress automatic migrations on startup
* Added option `OIDC_LOGOUT_URI` to trigger logout in OIDC on logout in Outline by @Shuttleu in https://github.com/outline/outline/pull/6539
* Added option `OIDC_DISABLE_REDIRECT` to prevent automatic OIDC redirect by @Shuttleu in https://github.com/outline/outline/pull/6544
* `MAXIMUM_IMPORT_SIZE` was deprecated and split into `FILE_STORAGE_IMPORT_MAX_SIZE` (for documents) and `FILE_STORAGE_WORKSPACE_IMPORT_MAX_SIZE` (for workspace imports) in https://github.com/outline/outline/pull/6566

### Fixes

* Fixed new paragraphs are lost in table cells imported from HTML
* Fixed an issue where tables with empty cells could fail to import from HTML
* Fixed an issue where quoted search queries with colons could cause a server error
* Fixed regression in styling of search filter options
* Hide image drag handles, empty captions, and selected outlines when printing in https://github.com/outline/outline/pull/6550
* Missing highlighted context on search results page in https://github.com/outline/outline/pull/6549
* Error for self-host draw.io by @TaoChan1005 in https://github.com/outline/outline/pull/6554
* Propagate Enter key events in `SuggestionsMenu` allows pressing `Enter` after typed url in embed input by @superkelvint in https://github.com/outline/outline/pull/6556
* Do not load `opensearch.xml` from CDN, so it uses the correct subdomain by @waf in https://github.com/outline/outline/pull/6567
* `CMD+Z` now works now works when the editor component does not technically have focus
* Served HTML `lang` attribute now reflects installation default language

## New Contributors
* @Shuttleu made their first contribution in https://github.com/outline/outline/pull/6539
* @TaoChan1005 made their first contribution in https://github.com/outline/outline/pull/6554
* @superkelvint made their first contribution in https://github.com/outline/outline/pull/6556
* @waf made their first contribution in https://github.com/outline/outline/pull/6567

**Full Changelog**: https://github.com/outline/outline/compare/v0.75.0...v0.75.1

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.75.1'>v0.75.1</a>.</em>

**Comment by alkanna** [comment]
Hey!

Is there any breaking change?
Upgrading the image does not work seamlessly:

```Defaulted container "outline" out of: outline, cloudsql-proxy
yarn run v1.22.19
$ node ./build/server/index.js
{"label":"lifecycle","level":"info","message":"Note: Restricting process count to 1 due to use of collaborative service"}
(node:28) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
{"level":"warn","message":"Environment configuration is invalid, please check the following:\n\n"}
{"level":"warn","message":"- MAXIMUM_IMPORT_SIZE must be a number conforming to the specified constraints"}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.```

**Reply by tommoor** [reply]
It's a bug, see: https://github.com/outline/outline/issues/6571#issuecomment-1956808306
